dev_guide/builds.adoc

```diff
@@ -818,70 +817,34 @@ endif::[]
 
 // end::using-a-proxy-for-git-cloning-1[]
 
-[[trusted-certificate-authorities]]
-==== Trusted Certificate Authorities
-
-The set of TLS certificate authorities that are trusted during a `git clone`
-operation are built into the {product-title} infrastructure images.  If your Git
-server uses a self-signed certificate or one signed by an authority not trusted
-by the image, you can disable TLS  verification by setting the
-`*GIT_SSL_NO_VERIFY*` environment variable to *true* in the appropriate strategy
-section of your build configuration. You can use the
-xref:../dev_guide/environment_variables.adoc#overview[`oc set env`] command to
-manage `*BuildConfig*` environment variables.
-
-[[using-private-repositories-for-builds]]
-==== Using Private Repositories for Builds
+[[source-secrets]]
+==== Source Secrets
 
-Supply valid credentials to build an application from a private repository.
+[[overview]]
+===== Overview
 
-Currently two types of authentication are supported: basic username-password
-and SSH key based authentication.
+Source secrets are used to provide the builder pod with access to git repositories
+that it would not normally have access to such as private repositories or
+repositories with self-signed/untrusted SSL certificates.
 
-[[basic-authentication]]
-===== Basic Authentication
-
-Basic authentication requires either a combination of `username` and `password`,
-or a `token` to authenticate against the SCM server. A `CA certificate` file,
-or a `.gitconfig` file can be attached.
-
-A xref:../dev_guide/secrets.adoc#dev-guide-secrets[`*secret*`] is used to store your keys.
-
-. Create the `*secret*` first before using the username and password to access
-the private repository:
-+
-====
-----
-$ oc secrets new-basicauth basicsecret --username=USERNAME --password=PASSWORD
-----
-====
+The following source secret configurations are supported:
 
-.. To create a Basic Authentication Secret with a token:
-+
-====
-----
-$ oc secrets new-basicauth basicsecret --password=TOKEN
-----
-====
+- xref:gitconfig-file[Gitconfig File]
+- xref:basic-authentication[Basic Authentication]
+- xref:ssh-key-authentication[SSH Key Authentication]
+- xref:trusted-certificate-authorities[Trusted Certificate Authorities]
 
-.. To create a Basic Authentication Secret with a CA certificate file:
-+
+[NOTE]
 ====
-----
-$ oc secrets new-basicauth basicsecret --username=USERNAME --password=PASSWORD --ca-cert=FILENAME
-----
+You can also use xref:combinations[combinations] of the these configurations
```

Comment:
OK， I have provided a PR to fix this: https://github.com/openshift/openshift-docs/pull/4549